courses/machine_learning/deepdive2/structured/solutions/3_automl_tables_babyweight.ipynb

```diff
@@ -0,0 +1,487 @@
+{
```

Comment:
<p>I let this run overnight, and it still isn't done yet. How long did this step take for you?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>It all depends what you put for your maximum node hours training budget. The max is 72 but obviously that takes a long time (took me like 1.5 days) but I tried it again with like 3 and it was done in a couple hours. So the longer someone is comfortable waiting, the bigger the budget they should allocate, at least in the hopes that longer training time will result in a better model.</p>

Reply:
Ah, I see. Could you please 1) Add these time estimates to the lab and 2) Give suggestions on what students can do in the meantime (like maybe start working on the next lab)? One of our most common pieces of feedback from students is that they don't like to wait around for this stuff to complete.

Reply:
Done. Ya that is exactly why I put the AutoML lab where I did. It is right after the lab where we generate the augmented data and before a bunch of BQML and Keras labs that they can work on while the AutoML training runs.

Reply:
Looks better, Ryan. Could you also suggest a minimum budget for this dataset? Sorry for the wait on this review, mine is still training :)

Reply:
<p>Done</p>